project.php

```diff
@@ -1238,20 +1238,20 @@ function do_history()
                     // Maybe move this array to Project.inc
                     $label_for_project_field_ = [
                         'deletion_reason' => _("Reason for Deletion"),
-                        'nameofwork' => _("Name of Work"),
-                        'authorsname' => _("Author's Name"),
+                        'nameofwork' => _("Title"),
+                        'authorsname' => _("Author"),
                         'projectmanager' => _("Project Manager"),
                         'language' => _("Language"),
                         'genre' => _("Genre"),
-                        'difficulty_level' => _("Difficulty Level"),
+                        'difficulty_level' => _("Difficulty"),
                         'special_code' => _("Special Day"),
                         'checkedoutby' => _("PPer/PPVer"),
                         'image_source' => _("Original Image Source"),
                         'image_preparer' => _("Image Preparer"),
                         'text_preparer' => _("Text Preparer"),
                         'extra_credits' => _("Extra Credits"),
                         'scannercredit' => _("Scanner Credit"),
-                        'clearance' => _("Clearance Information"),
+                        'clearance' => _("Clearance Line"),
                         'postednum' => _("Posted Number"),
```

Comment:
you changed this to "PG etext number" below. Which one should we settle on or are these different?

Reply:
Thanks for catching that! They are the same and I went with "PG etext number" since that's what on the project and project search pages.